src/main/nvidiaSmi.ts

```diff
@@ -72,6 +75,11 @@ const getNvidiaSmiCommand = async (): Promise<string | undefined> => {
         if (targetDir) {
             return path.join(basePath, targetDir, 'nvidia-smi.exe');
         }
+        // check if smi is in System32
+        const smiPath = `${WINDIR}\\System32\\nvidia-smi.exe`;
+        if ((await fs.stat(smiPath)).isFile()) {
```

Comment:
Doesn't `fs.stat` throw an error if the file/dir doesn't exist? Is this function intended to throw an error?

You might want to use our `checkFileExists` function.

Reply:
It is intended for that